socorro/unittest/external/es/test_supersearch.py

```diff
@@ -1435,11 +1435,19 @@ def test_get_with_beta_version(self):
             assert "4.0b" in hit["version"]
 
     def test_get_against_nonexistent_index(self):
+        # Create start and end dates to look at indices in, but anchor them to monday so
+        # it's not moving around over week barriers
+        end = datetimeutil.utc_now()
+        while end.weekday() != 0:
+            end = end - datetime.timedelta(days=1)
+        start = end - datetime.timedelta(days=10)
+        end = end.strftime("%Y-%m-%dT%H:%M:%S")
+        start = start.strftime("%Y-%m-%dT%H:%M:%S")
         config = self.get_base_config(
             cls=SuperSearchWithFields, es_index="socorro_test_reports_%W"
         )
         api = SuperSearchWithFields(config=config)
-        params = {"date": [">2000-01-01T00:00:00", "<2000-01-10T00:00:00"]}
+        params = {"date": [">%s" % start, "<%s" % end]}
```

Comment:
With the new code, indices between 2000-01-01 and 2000-01-10 would get dropped since they're _way_ before the retention policy.